### ISSUE TITLE
More flexibility in nested field conditions

### ISSUE THREAD

**Issue by stuartcusackie** [statamic/ideas#783]
### Bug description

**This probably isn't a bug. Let me know if I should move it to ideas.**

I have a complicated page builder replicator:

- Sections (replicator)
  - Full Section (fieldset)
    - Blocks (replicator)
      - Card Group (fieldset) 
        - Style (field)
        - Items (replicator)
          - Link (field)
          - Icon (field)

I am trying to use field conditions to toggle the card_group.items fields, like so:

```
handle: link
if:
  card_group.style: 'linked'

handle: icon
if:
  card_group.style: 'iconed'
```

The above doesn't work. I have also tried `root.style` but I think root targets the top-level 'Sections' replicator.

### How to reproduce

See above.

### Versions

Statamic 3.3.7 Solo
Laravel 8.83.8
PHP 8.1.1
edalzell/blade-directives 3.7

### Installation

Fresh statamic/statamic site via CLI

**Comment by jasonvarga** [comment]
A `parent` seems like the right move.

**Comment by stuartcusackie** [comment]
> A `parent` seems like the right move.

I was thinking that if it's possible then it would be great to have full access to the the ancestor tree. With 'parent' you wouldn't be able to access any grandparents, example in my case:

```
handle: icon
if:
grid.style: 'equals columns'
section.style: 'equals large'
```

If I remember correctly, the ACF plugin for WordPress does something like this and it is useful sometimes.

**Comment by jasonvarga** [comment]
Using the handle to mean any field at any position won't really work. You could easily have multiple fields named the same at multiple positions. For instance, within grid rows, or replicator sets.

**Comment by stuartcusackie** [comment]
You're probably right but I was thinking that every field has a parent, whether it's a replicator, grid, or a bard, and that it should be theoretically possible to target any ancestor all the way up the tree since we know the field handles of these parents. It would be a funky piece of JavaScript if it were possible.

Maybe I shouldn't be nesting fields so much though. 'parent' and 'root' should be enough in most cases.

**Comment by florianbrinkmann** [comment]
I’m having a similar issue in a project currently, and a `parent` in addition to `root` would be really cool :)

**Comment by florianbrinkmann** [comment]
I worked on a way to add a `parent`, and created a pull request: https://github.com/statamic/cms/pull/9385 
Maybe you could have a look, @jasonvarga :)

**Comment by jasonvarga** [comment]
Closed by statamic/cms#9385

**Comment by jesseleite** [comment]
Just added documentation as well here https://github.com/statamic/docs/pull/1502 👍
